### PR TITLE
fix: prompt SvelteKit 2 migration during Svelte 5 migration if necessary

### PR DESCRIPTION
**Pull request by dummdidumm**
closes https://github.com/sveltejs/svelte/issues/13462

### CODE REVIEW

File: packages/migrate/migrations/svelte-5/index.js [hunk 1]
```diff
@@ -56,6 +58,42 @@ export async function migrate() {
 		}
 	}
 
+	const kit_dep = pkg.devDependencies?.['@sveltejs/kit'] ?? pkg.dependencies?.['@sveltejs/kit'];
+	if (kit_dep && semver.validRange(kit_dep) && semver.gtr('2.0.0', kit_dep)) {
```
Comment (benmccann) [inline packages/migrate/migrations/svelte-5/index.js:62]: it's a little surprising to me to see `2 > kit_dep` vs `kit_dep < 2`

File: packages/migrate/migrations/svelte-5/index.js [hunk 2]
```diff
@@ -56,6 +58,42 @@ export async function migrate() {
...
+			colors
+				.bold()
+				.yellow(
+					'\nDetected SvelteKit 1. You need to upgrade to SvelteKit version 2 first (`npx svelte-migrate sveltekit-2`).\n'
+				)
+		);
+		const response = await prompts({
+			type: 'confirm',
+			name: 'value',
+			message: 'Run `sveltekit-2` migration now?',
```
Comment (benmccann) [inline packages/migrate/migrations/svelte-5/index.js:73]: I'm not sure it makes sense to print backticks to the terminal?
```suggestion
			message: 'Run sveltekit-2 migration now?',
```

**Review by benmccann — COMMENTED** [review]
personally I would just bail if SvelteKit 1 is detected rather than trying to run both migrations because I think it would be good for the user to run the SvelteKit 2 migration and then commit the result to git before trying to run the next migration

**Comment by dominikg** [comment]
agree with Ben, just log the command for the migration and ask them to do that first, commit and TEST the result before starting with svelte5 migration.

otherwise we'll get a few strange bug reports regarding svelte5 migration that are really about kit

File: packages/migrate/migrations/svelte-5/index.js (hunk 1 above)
Comment (dummdidumm) [reply packages/migrate/migrations/svelte-5/index.js:58]: yoda-speak - it's done like that in all other migration, I don't care honestly.

**Review by benmccann — APPROVED** [review]

**Pull request merged**